6.X/6.9.0/docs/01-command_line_arguments.md

```diff
@@ -0,0 +1,332 @@
+<!-- Space: TOOLS -->
+<!-- Parent: Installer -->
+<!-- Title: Command Line Arguments -->
+<!-- Layout: plain -->
+
+# Command Line Arguments
+
+<br/>
+
+## Command: `deploy`
+
+`--skip-namespace`
+
+- installer does not deploy the `namespace.yaml` manifest.
+  It expects the Namespace to exist and to match the value in `values.yaml`
+  If there is a mismatch, the installer will fail as no validation is in place.
+
+`--skip-pull-secret`
+
+- the services expect the pull secret to exist, to have the expected name (`sysdigcloud-pull-secret`) and to allow access to the registry.
+  to have the expected name (`sysdigcloud-pull-secret`) and to allow access to the registry.
+- if the pull secret is missing, the behaviour could be unpredictable:
+  some Pods could start if they can find the image locally and if their `imagePullPolicy`
+  is not `Always`
+- Other Pods will fail because they can't pull the image
+
+`--skip-serviceaccount`
+
+- The user must provide SAs with the exact same name expected:
+
+```text
+sysdig-serviceaccount.yaml:  name: sysdig
+sysdig-serviceaccount.yaml:  name: node-labels-to-files
+sysdig-serviceaccount.yaml:  name: sysdig-with-root
+sysdig-serviceaccount.yaml:  name: sysdig-elasticsearch
+sysdig-serviceaccount.yaml:  name: sysdig-cassandra
+```
+
+- One implication of this is that unless the `node-to-labels` SA is added,
```

Comment:
Please avoid using abbreviation, such as SA

Reply:
We are getting the information from installer repo:
- https://github.com/draios/installer/tree/master/docs
- https://github.com/draios/installer/tree/master/examples
- https://github.com/draios/installer/blob/master/README.md

It would be good for the next release, the documentation team to be involved in the early stages to fix the source.

@bennythejudge @pravin-jothinathan @figarocorso